.flowconfig

```diff
@@ -39,7 +39,6 @@ module.file_ext=.js
 module.file_ext=.jsx
 module.file_ext=.json
 
-module.name_mapper='^image![a-zA-Z0-9$_-]+$' -> 'GlobalImageStub'
```

Comment:
They removed this because the `require('./file.png!image')` syntax has been deprecated since around rn@0.7? something really early on.